cfgov/jinja2/v1/owning-a-home/mortgage-closing/index.html

```diff
@@ -0,0 +1,162 @@
+      <p class="u-mb30">
+        <a href="/askcfpb/1917/what-should-i-do-if-i-find-an-error-in-one-of-my-mortgage-closing-documents.html"
+           class="a-link
+                  a-link__jump">
+          <span class="a-link_text">Learn more about what to do if you find an error</span>
+        </a>
+      </p>
+      {% include "_layouts/ratings-form.html" %}
+    </section>
+    <!-- .content_main -->
+    <aside class="content_sidebar">
+      <div class="block
+                  block__flush-top block__padded-top">
+        {{ social_media.render( {
+              "twitter_text": "About to close on a mortgage? The @CFPB’s tools and checklist help you close with confidence.",
+              "email_title": "About to close on a mortgage?",
+              "email_text": "The @CFPB’s tools and checklist help you close with confidence.",
+              "email_signature": "-- From the CFPB",
+              "linkedin_title": "About to close on a mortgage?",
+              "linkedin_text": "The @CFPB’s tools and checklist help you close with confidence. Via @CFPB"
+        } ) }}
+      </div>
+      <section class="block u-mt0">
+        <header class="m-slug-header">
+            <h2 class="a-heading">
+                Resources
+            </h2>
+        </header>
+        <ul class="m-list m-list__spaced m-list__unstyled">
+          <li class="m-list_item">
+            <a class="a-link a-link__jump" href="/owning-a-home/">
+              <span class="a-link_text">Owning a Home: Tools and resources for home buyers</span>
+            </a>
+          </li>
+          <li class="m-list_item">
+            <a class="a-link
+                      a-link__jump
+                      a-link__icon
+                      a-link__no-wrap
+                      cf-icon
+                      cf-icon__after
+                      cf-icon-download"
+               href="http://www.consumerfinance.gov/f/201503_cfpb_your-home-loan-toolkit-web.pdf">
+              <span class="a-link_text">Your home loan toolkit: A step-by-step guide</span>
+            </a>
+          </li>
+          <li class="m-list_item">
+            <a class="a-link a-link__jump"
+               href="/find-a-housing-counselor/">
+              <span class="a-link_text">Find a housing counselor</span>
+            </a>
+          </li>
+          <li class="m-list_item">
+            <a class="a-link a-link__jump" href="/complaint/#mortgage">
+              <span class="a-link_text">Submit a complaint about problems with your mortgage lender</span>
+            </a>
+          </li>
+        </ul>
+      </section>
+      <section class="block u-mt0">
+        <header class="m-slug-header">
+            <h2 class="a-heading">
+                For industry professionals
+            </h2>
+        </header>
+        <ul class="m-list m-list__spaced m-list__unstyled">
+          <li class="a-list_item">
+            <span class="jump-link_intro">Mortgage professionals</span><br>
```

Comment:
Removed in https://github.com/cfpb/cfgov-refresh/pull/3165/commits/adea54f8ecb4f983b08882aaf842c2e812e402bf.